### DISCUSSION TITLE
iOS offline raster maps download option

### DISCUSSION THREAD

**Discussion by Andre1617** [osmandapp/OsmAnd-iOS#5009] [Ideas]
Hello I’ve been using for sometime the android version and now I’m back on iPhone and I realized there’s no way to download the Microsoft Earth maps right from the app on iOS.
Could we get raster map downloads do work the same way as vector maps are downloaded, so I can just go choose country or county of country and just download the Microsoft imagery for that area, also there should be nicely distinct borders of the both vector and Microsoft imagery maps so you can see where your downloaded map ends and make it more clear in settings to be able to clear the cashed map so I can just go clear the cashed map and just use what I downloaded

who else agrees ?

**Comment by DmitryAlexei** [comment]
Have you tried to download raster maps as described in the user guide?
Online maps for a single country may take gigabytes of storage
https://osmand.net/docs/user/map/raster-maps#download--update-tiles

**Comment by Definence** [comment]
I tried to follow this guide, but it seems that options like "Download map" or "Update map" only appear when an online map is set as the default on IOS. When "Touring view" is the default map and Microsoft Earth is used as the outer layer, these options simply don’t show up.
That's why he probably wasn't able to do so.
If you select Microsoft Earth as a default map, download the raster map, and then switch back to Touring view as a default, the outer layer of Microsoft Earth will work from the offline source.
